BlueprintUICommonControls/Sources/TextField.swift

```diff
@@ -10,6 +10,9 @@ public struct TextField: Element {
     public var onChange: ((String) -> Void)? = nil
     public var secure: Bool = false
     public var isEnabled: Bool = true
+    
+    public var font : UIFont? = nil
```

Comment:
These are nil so we can preserve the default color and font, and matches the properties themselves on UITextField.